website/docs/components/link/standalone/partials/guidelines/guidelines.md

```diff
@@ -3,105 +3,104 @@
+Generally, we recommend `arrow-right` for internal links and `external-link` (or similar) for external links. 
 
+<Doc::Layout @spacing="12px">
   <Hds::Link::Standalone @text="View billing" @icon="arrow-right" @iconPosition="trailing" @color="primary" @route="components" />
   <Hds::Link::Standalone @text="Submit feedback" @icon="external-link" @iconPosition="trailing" @color="primary" @route="components" />
   <Hds::Link::Standalone @text="Consul documentation" @icon="docs-link" @iconPosition="trailing" @color="primary" @route="components" />
   <Hds::Link::Standalone @text="Service mesh tutorial" @icon="learn-link" @iconPosition="trailing" @color="primary" @route="components" />
+</Doc::Layout>
 
+<Doc::Layout @spacing="12px">
   <Hds::Link::Standalone @text="View billing" @icon="arrow-right" @iconPosition="trailing" @color="secondary" @route="components" />
   <Hds::Link::Standalone @text="Submit feedback" @icon="external-link" @iconPosition="trailing" @color="secondary" @route="components" />
   <Hds::Link::Standalone @text="Consul documentation" @icon="docs-link" @iconPosition="trailing" @color="secondary" @route="components" />
   <Hds::Link::Standalone @text="Service mesh tutorial" @icon="learn-link" @iconPosition="trailing" @color="secondary" @route="components" />
-
-!!! Info
-
-If adding a specific icon is relevant to the user experience, and none of the icons available in Flight fit your use case, consider [requesting a custom icon](https://github.com/hashicorp/design-system/issues/new/choose).
-!!!
+</Doc::Layout>
 
 ## Common patterns
 
 ### Within button sets
 
-**Link (Standalone)** can commonly be seen within **button sets**.
+Standalone Links often appear in [Button Sets](/components/button-set).
 
 <Hds::ButtonSet>
   <Hds::Link::Standalone @text="Previous" @icon="chevron-left" @iconPosition="leading" @color="primary" @route="components" />
   <Hds::Button @text="Submit" @color="primary" />
   <Hds::Button @text="Cancel" @color="secondary" />
 </Hds::ButtonSet>
 
-When included in a ButtonSet, Link (Standalone) is generally used for navigating between steps in a multi-page form (ie. directing the user back to the previous step) or for linking to external documentation. As such, they should be displayed as the right most button in the row.
-
 ## Content
 
-- Text should be short and to the point (~25 characters). Text labels should not consist of full sentences, but should provide enough context to be useful.
-- Language should be used consistently within each product (ie. when using "Edit" on one page, use that same convention throughout the rest of the application, not "Change").
+- Text should be short and to the point (~25 characters). Text labels should not consist of full sentences but should provide enough context to be useful.
+- Use language consistently within each product (e.g., when using “Edit” on one page, use that same convention throughout the rest of the application, not "Change").
```

Comment:
```suggestion
- Use language consistently within each product (e.g., if “Edit” is used on one page, use that same convention throughout, instead of a different similar word such as "Change").
```